docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -10,6 +10,9 @@ The following figure illustrates flex container terminology and alignment axes,
 
 ![Flex container terminology](media/flex-container.jpg)
 
+> [!NOTE]
+> FlexContainer class is deprecated in API level 8 and will be removed in API level 10. Please use FlexLayout instead!
```

Comment:
```suggestion
> Use FlexLayout instead of FlexContainer class. FlexLayout class is deprecated since Tizen 6.0 and will be supported for subsequent two releases only .
```

Just aligning the comeent with @mijin85cho comment https://github.com/Samsung/tizen-docs/pull/1272/files#r499959363